lib/simulator-xcode-6.js

```diff
@@ -185,6 +185,24 @@ class SimulatorXcode6 extends EventEmitter {
     return appDirs.length !== 0;
   }
 
+  /**
+   * Returns user installed bundle ids which has 'bundleName' in their Info.Plist as 'CFBundleName'
+   * @param {string} bundleId - The bundle id of the application to be checked.
+   */
+  async getUserInstalledBundleIdsByBundleName (bundleName) {
```

Comment:
can we have a test for this method?

Reply:
Yes, I'm adding a case in simulator-e2e-specs